packages/atomic/src/components/ipx/atomic-ipx-tabs/atomic-ipx-tabs.tsx

```diff
@@ -0,0 +1,33 @@
+import {Component, Element, h} from '@stencil/core';
+
+/**
+ * @internal
+ */
+@Component({
+  tag: 'atomic-ipx-tabs',
+  styleUrl: './atomic-ipx-tabs.pcss',
+  shadow: true,
+})
+export class AtomicIPXTabs {
```

Comment:
Mini truc mais le nom ressemble tellement a l'autre component j'aurais mis IPXTabsContainer ou qqchose lol c'est comme ca pour insight panel aussi j'imagine?

Reply:
Les components ont toujours un nom commencçant avec `AtomicXXXX`. 
J'ai juste gardé ce format comme celui du Insight.

Reply:
Just to avoid confusion if you see other components which aren't prefixed with `Atomic`:
* We prefix custom elements (such as this one) with `Atomic`
* We don't prefix functional components (such as `Button`) with anything